citizen.ts

```diff
@@ -1,283 +1,138 @@
-      // concatenated contract source code and the contract ID.
-      .map((contractName) =>
-        buildRendezvousData(deploymentPlan, contractName, manifestDir)
-      )
-      // Use the contract ID as a key, mapping to the concatenated contract
-      // source code.
-      .map((rendezvousContractData) => [
-        rendezvousContractData.rendezvousContractId,
-        rendezvousContractData.rendezvousSourceCode,
-      ])
-  );
-
-  const clarinetToml = toml.parse(
+  const parsedManifest = toml.parse(
     readFileSync(manifestPath, { encoding: "utf-8" })
   ) as any;
-  const cacheDir = clarinetToml.project?.cache_dir || "./.cache";
+  const cacheDir = parsedManifest.project?.["cache_dir"] ?? "./.cache";
 
-  // Deploy the contracts to the empty simnet session in the correct order.
-  await deployContracts(
-    simnet,
-    sortedContractsByEpoch,
-    manifestDir,
+  const rendezvousData = buildRendezvousData(
     cacheDir,
-    (name, sender, props) =>
-      getContractSource(
-        [sutContractName],
-        rendezvousSources,
-        name,
-        sender,
-        props,
-        manifestDir
-      )
-  );
-
-  // Filter out addresses with zero balance. They do not need to be restored.
-  const sbtcBalancesToRestore: Map<string, number> = new Map(
-    [...sbtcBalancesMap.entries()].filter(([_, balance]) => balance !== 0)
+    deploymentPlan,
+    sutContractName,
+    manifestDir
   );
 
-  // After all the contracts and requirements are deployed, if the test wallets
-  // had sBTC balances previously, restore them. If no test wallet previously
-  // owned sBTC, skip this step.
-  if ([...sbtcBalancesToRestore.keys()].length > 0) {
-    restoreSbtcBalances(simnet, sbtcBalancesToRestore);
-  }
-
-  return simnet;
-};
-
-/**
- * Groups contracts by epoch from the deployment plan.
- * @param deploymentPlan The parsed deployment plan.
- * @returns A record of contracts grouped by epoch. The record key is the epoch
- * string, and the value is an array of contracts. Each contract is represented
- * as a record with the contract name as the key and a record containing the
- * contract path and clarity version as the value.
- */
-export const groupContractsByEpochFromDeploymentPlan = (
-  deploymentPlan: DeploymentPlan
-): ContractsByEpoch => {
-  return deploymentPlan.plan.batches.reduce(
-    (acc: ContractsByEpoch, batch: Batch) => {
-      const epoch = batch.epoch;
-      const contracts = batch.transactions
-        .filter((tx) => tx["emulated-contract-publish"])
-        .map((tx) => {
-          const contract = tx["emulated-contract-publish"]!;
-          return {
-            [contract["contract-name"]]: {
-              path: contract.path,
-              clarity_version: contract["clarity-version"],
-            },
-          };
-        });
-
-      if (contracts.length > 0) {
-        acc[epoch] = (acc[epoch] || []).concat(contracts);
-      }
+  const sessionId = `${Date.now()}`;
```

Comment:
Why not use proper unique IDs?
```suggestion
  const sessionId = randomUUID();
```